### PR TITLE
Expandible region anno for variants with many genes

### LINKED ISSUE #3554: Make variant region anno expandible, just like functional anno in variants list
Regional anno is the column on the left. Instead of 3 genes show at least 30, same as for funcional anno (second col from left)

image

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3554 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. Check SVs and cancer SVs pages and make sure that when the variant has more than 3 genes, the region anno are shown for all of them (they'r max 30 anyway)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested on stage: 🆗 


Before:

image


After:

image

File: scout/server/blueprints/variants/templates/variants/components.html [hunk 1]
```diff
@@ -40,6 +40,25 @@
   </div>
 {% endmacro %}
 
+{% macro variant_region_anno_cell(variant) %}
+  <div class="d-flex justify-content-between align-items-center">
+    <div>
+      {% if variant.functional_annotations|length >= 5 %}
+        <a class="mr-3" data-bs-toggle="collapse" href="#_regionanno_{{variant._id}}" role="button" aria-expanded="false" aria-controls="_{{variant._id}}">{% if variant.missing_data %}[first 30 genes]{% else %}[...]{% endif %}</a>
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/components.html:47]: 👍

File: scout/server/blueprints/variants/templates/variants/components.html [hunk 2]
```diff
@@ -40,6 +40,25 @@
   </div>
 {% endmacro %}
 
+{% macro variant_region_anno_cell(variant) %}
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/components.html:43]: 👍

**Review by dnil — APPROVED** [review]
Nice!

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [4.58.1]
+### Changed
```
Comment (northwestwitch) [inline CHANGELOG.md:8]: Oh no, I messed up with the changelog, sorry. Ket's fix in the next PR we merge!

Comment (dnil) [reply CHANGELOG.md:8]: He, too bad I didn't read this: I added to the confusion by leaving the SRS part on the 4.58.1. Good that we don't release on merge. 😄

**Pull request merged**